src/pages/docs/matchmaker/concepts/lobby-tags.mdx

```diff
@@ -0,0 +1,61 @@
+# Lobby tags
+
+Lobby tags are an optional property that can be passed to the [`lobbies.find`](/docs/matchmaker/api/lobbies/find), [`lobbies.join`](/docs/matchmaker/api/lobbies/join), and [`lobbies.create`](/docs/matchmaker/api/lobbies/create) endpoints to allow filtering which lobbies the matchmaker will route a player to.
+
+To enable tags, set [`taggable`](http://localhost:3000/docs/general/config#matchmaker-game-modes-game-mode-taggable) to true.
+
+When passed to [`lobbies.create`](/docs/matchmaker/api/lobbies/create), the newly created lobby will have the given tags.
```

Comment:
I think this needs to clarify lobbies.find will set the tags too, right? I know you go in to this below in the example, but thrown off by this.